packages/format-library/src/link/index.native.js

```diff
@@ -108,10 +110,25 @@ export const link = {
 			speak( __( 'Link removed.' ), 'assertive' );
 		}
 
+		getURLFromClipboard = async () => {
```

Comment:
[The build complains](https://travis-ci.com/WordPress/gutenberg/jobs/252800722#L680) that Class Properties is not enabled so, let's change this to:
```suggestion
		async getURLFromClipboard()
```